modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -110,6 +125,270 @@ export class CosmosCoin extends BaseCoin {
     throw new Error('Method not implemented.');
   }
 
+  /**
+   * Builds a funds recovery transaction without BitGo
+   * @param {RecoveryOptions} params parameters needed to construct and
+   * (maybe) sign the transaction
+   *
+   * @returns {AtomTx} the serialized transaction hex string and index
+   * of the address being swept
+   */
+  async recover(params: RecoveryOptions): Promise<CosmosLikeCoinRecoveryOutput> {
+    // Step 1: Check if params contains the required parameters
+    if (!params.bitgoKey) {
+      throw new Error('missing bitgoKey');
+    }
+
+    if (!params.recoveryDestination || !this.isValidAddress(params.recoveryDestination)) {
+      throw new Error('invalid recoveryDestination');
+    }
+
+    if (!params.userKey) {
+      throw new Error('missing userKey');
+    }
+
+    if (!params.backupKey) {
+      throw new Error('missing backupKey');
+    }
+
+    if (!params.walletPassphrase) {
+      throw new Error('missing wallet passphrase');
+    }
+
+    // Step 2: Fetch the bitgo key from params
+    const bitgoKey = params.bitgoKey.replace(/\s/g, '');
+
+    // Step 3: Instantiate the ECDSA signer and fetch the address details
+    const MPC = new Ecdsa();
+    const chainId = await this.getChainId();
+    const currPath = 'm/0';
```

Comment:
Please define this as a constant somewhere and export it. The SDK and WP should be alligned on what the base path is, and the easiest way to do that is to use the exported SDK value.

Reply:
added

Reply:
nit: If we are using currPath as 'm/0'
Then we dont need 'Starting Scan Index' and 'Address Scanning Factor' field in WRW form.
As we have restricted it to 0 in the path.

Not a blocker for this PR.